.circleci/config.yml

```diff
@@ -14,12 +14,9 @@ jobs:
       - run:
           name: "Test: Build and Spin-up Django API Service"
           command: "cd tdrs-backend; docker-compose up -d --build"
-      - run:
-          name: "Tear Down Django API Service"
```

Comment:
This step seems to be unnecessary and was causing intermittent connection failures since it was tearing down `clamav-rest` which is needed by the next step. While `docker-compose run` will pull up any dependent containers it wasn't giving clamav enough time to complete its startup before running the tests. By removing this tear down step we ensure that clamav-rest will be accessible by the time we run the tests.